### PR TITLE
Update Zichen's PPP

### PR DESCRIPTION
**Pull request by zichen-3974**
[link](https://zichen-3974.github.io/tp/team/zichen-3974.html)

### CODE REVIEW

File: docs/team/zichen-3974.md [hunk 1]
```diff
@@ -3,35 +3,49 @@ layout: page
...
 
 
-* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s2.github.io/tp-dashboard/?search=zichen-3974&breakdown=true)
+* **Enhancements to existing features**:
+  * Refactor UI (Pull requests [#58](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/58), [#78](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/78), [#178](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/178), [#228](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/228), [#247](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/247))
+    * Restructure UI to include new components such as the scheduled meet up pane, the meet recommendation pane, the user profile pane and the sidebar. 
+    * Modify help window's UI to include summary of command usage.
+    * Updated the UI colour scheme.
+    * Designed background for side panel with Figma.
+    * Replaced menu bar with sidebar and `File`,`Help` options replaced with intuitive icon buttons.
```
Comment (mynameizzhafeez) [inline docs/team/zichen-3974.md:24]:
```suggestion
    * Replaced menu bar with sidebar and replaced `File`,`Help` options with intuitive icon buttons.
```

The phrase structure felt a little weird.

File: docs/team/zichen-3974.md [hunk 2]
```diff
@@ -3,35 +3,49 @@ layout: page
...
+    * Modified UI to include various visual effects using JavaFX and CSS, such as animated scaling, button rotation, and color changes when hovering or clicking over different components; providing users with a more dynamic and engaging experience.
```
Comment (mynameizzhafeez) [inline docs/team/zichen-3974.md:25]:
```suggestion
    * Modified UI to include various visual effects using JavaFX and CSS, such as animated scaling, button rotation, and color changes when hovering or clicking over different components, providing users with a more dynamic and engaging experience.
```

I think semicolons are used when the next part could be a sentence on its own. But in this case, the second part does not make sense as its own sentence.

**Review by mynameizzhafeez — APPROVED** [review]
LGTM. Just see the minor changes.

File: docs/team/zichen-3974.md [hunk 3]
```diff
@@ -3,35 +3,49 @@ layout: page
 title: Zichen's Project Portfolio Page
 ---
 
-### Project: eduMate
+### Project: EduMate
 
-eduMate is a desktop app designed for NUS students to manage their academic and social lives by lowering the barriers to meet up and make new friends within modules and school. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+EduMate is a desktop app designed for NUS students to manage their academic and social lives by lowering the barriers to meet up and make new friends within modules and school. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
```
Comment (mynameizzhafeez) [inline docs/team/zichen-3974.md:8]:
```suggestion
EduMate is a desktop app designed for NUS students to manage their academic and social lives by lowering the barriers to meet up and make new friends within modules and school. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 30 kLoC.
```

File: docs/team/zichen-3974.md [hunk 4]
```diff
@@ -3,35 +3,49 @@ layout: page
...
+  * What it does: allows the user to organise a customised meeting or choose a meet up recommendation generated by the `Meet` command. The user can remove any organised meetings with the `Unorganise` command.
+  * Justification: This feature allows users to track and manage meet ups to keep their schedule organised.
+  * Highlights: This enhancement affects existing commands such as `Meet`, `Edit`, `Delete` and hence required additions to those commands.
 
 
-* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s2.github.io/tp-dashboard/?search=zichen-3974&breakdown=true)
+* **Enhancements to existing features**:
+  * Refactor UI (Pull requests [#58](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/58), [#78](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/78), [#178](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/178), [#228](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/228), [#247](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/247))
+    * Restructure UI to include new components such as the scheduled meet up pane, the meet recommendation pane, the user profile pane and the sidebar.
```
Comment (mynameizzhafeez) [inline docs/team/zichen-3974.md:20]: These could potentially have links to the file on GitHub; they are actually quite significant (and a lot of work too).

Comment (zichen-3974) [reply docs/team/zichen-3974.md:20]: Hmm makes sense, will add some. Thanks!

**Pull request merged**